_graphql/models/config/config.yml

```diff
@@ -0,0 +1,7 @@
+modelConfig:
+  DataObject:
+    operations:
+      read:
+        plugins:
+          filter: true
```

Comment:
This adds the ability to filter results by page.

I've disabled pagination for now to make things a bit simpler.